### PR TITLE
Create a desktop version from a mobile one. 

### PR DESCRIPTION
**Pull request by lojacho**
In this pull request i was able to do:

Create a desktop version from the past mobile version. 
Use flexbox and grid layout.
Use percentages in dimension in order to make it responsive. 
Use media queries for the desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,145 +22,161 @@
...
 			</ul>
 		</header>
-		<section class="headline">
-			<h1>Hey There. I’m Diego</h1>
-			<h2>I’am a Software Developer</h2>
-			<p>I can help you build a product , feature or website Look through some of my work and experience! If you
-				like
-				what you see and have a project you need coded,
-				don’t hestiate to contact me.
-			</p>
```
Comment (shahadat3669) [inline index.html:39]: Maybe you can put text in a single line like below:

```
<p>
I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
</p>
```

File: styles/desktop.css [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  }
+
+  .cardsGrid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 1.8em;
+  }
+
+
```
Comment (shahadat3669) [inline styles/desktop.css:198]: Maybe you can remove the extra white space.

File: styles/desktop.css [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    flex-direction: column;
+  }
+
+  .contentTop h2 {
+    margin-top: 0;
+    padding-top: 0;
+    margin-bottom: 1rem;
+    font-size: 2rem;
+    line-height: 4rem;
+
```
Comment (shahadat3669) [inline styles/desktop.css:167]: Maybe you can remove the extra white space.

File: styles/desktop.css (hunk 2 above)
Comment (chiztechnology) [inline styles/desktop.css:198]: It's recommende to keep only one blank line between the brackets to make the code clear and readable.

File: styles/desktop.css [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    flex-direction: column;
+    width: 100%;
+    height: auto;
+  }
+
+  .worksSection {
+    display: flex;
+    flex-direction: column;
+    width: 80%;
+    margin:auto;
```
Comment (shahadat3669) [inline styles/desktop.css:113]: maybe you should give some space after ":" like below:
`margin: auto;`

File: index.html [hunk 5]
```diff
@@ -21,145 +22,161 @@
...
+							industry's standard</p>
+						<ul>
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</section>
+					<footer class="footerCards">
+						<button type="button">See Project</button>
+					</footer>
```
Comment (luigirazum) [inline index.html:178]: Hello Diego, I would like to suggest you checking if we can use this footer inside de cards... as I thought it is for the footer of the page.

File: index.html [hunk 6]
```diff
@@ -216,14 +233,14 @@ <h1 class="formHeader">I'm always interested in hearing about new projects, so i
...
-							required>
+						<input class="colorbox" type="email" id="formMail" name="userEmail"
+							placeholder="    Email address" required>
 					</label>
 				</div>
 				<div class="message">
 					<label for="formMessage">
-						<textarea id="formMessage" name="text" maxlength="500"
-							placeholder="Enter text here" required></textarea>
+						<textarea id="formMessage" name="text" maxlength="500" placeholder="Enter text here"
```
Comment (chiztechnology) [inline index.html:242]: could you please set the value of name attribute to something more clear such as name="message" or name="content" to make the data collected in the form become more expicit and clear 🎊 ✔️

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Good work. But there is some issue with your linter.
image

**Review by chiztechnology — COMMENTED** [review]
It's amazing ! I reall appreciate the huge accomplishments you just made !
🎁 🎊 
There are some reviews according to the code you presented.
Please have a look and I hope it will help you improve your soft skills.

Kindest regards 🤝

**Pull request closed without merging**